README.md

```diff
@@ -26,34 +26,34 @@
 3. `yarn dev` to run the repo
 4. (optional) `yarn storybook` to
 
-## Install Cosmos-SDK
+## Install Regen Ledger
 
-`git clone https://github.com/cosmos/cosmos-sdk` and **within that directory**:
+`git clone https://github.com/regen-network/regen-ledger/` and **within that directory**:
 
-1. `git checkout v0.46.11`
-2. `make build`
+1. `make build` - build ledger locally
+2. `mv build/regen build/regenlocal` - rename to `regenlocal` to avoid conflicts with the `regen` binary
```

Comment:
Why is it necessary to have a separate local version of the binary?  If someone wants to use the existing `regen` binary in their path, shouldn't that be just fine?

I think the best approach is to either:
1. Install the latest verison of Regen binary (if not already installed), and ensure it is accessible in the $PATH
2. Use the already installed version of regen binary on your system

Any local configuration we want for testing should be doable by setting up a local network with its own `$REGEN_HOME`.  As long as the env variable of `$REGEN_HOME` is set in the context of testing the groups-ui, we should be able to have locally configured network for local testing that works fine.

Probably it makes sense to not store that home directory in the user's home dir (e.g. `~/.regenlocal`, but rather put it relative to this repo, and also include it in the .gitignore.

Adding something like `export REGEN_HOME=$(pwd)/.regen` to the makefile  should do the trick.

Reply:
> Why is it necessary to have a separate local version of the binary? If someone wants to use the existing regen binary in their path, shouldn't that be just fine?

I wasn't sure - tbh I'm not super familiar with the differences in how regen / cosmos operate locally vs when pointed at the network, and haven't worked a ton with makefiles / binaries in general, and this was mainly done for my own peace of mind 😄 

Everything you've said here makes sense - appreciate your feedback @clevinson , will update 👍

Reply:
Great! Lmk if anything I said was confusing or if you need to talk anything through further.

Reply:
@clevinson just updated, adding the makefile command `make install-local-ledger` which will clone and build regen ledger into a project subfolder to be used by the other make commands.

This should make setup easier, and avoid any potential conflicts with binaries. LMK how it looks